### PR TITLE
Added context to the backend components API.

### LINKED ISSUE #1104: Add more context information for the API references
- [ ] Provider 
* What is the order of the methods a user needs to call and why 
* What each method does?
- [ ] Aquarius

### PR DESCRIPTION
**Pull request by mariacarmina**
Fixes #1104 .

Changes proposed in this PR:

- updated `provider-api.md`;
- reorder endpoints to be clearer how to use;
- update aqua examples

### CODE REVIEW

**Comment by jamiehewitt15** [comment]
> Most of described aquarius endpoints are self explanatory. Do you have a suggestion for improving aquarius REST API doc, @jamiehewitt15?

The descriptions are ok but the examples could be added to or updated. Like for example, I just tried the first two examples and they both give error responses. It would be nicer to give examples that work

**Review by jamiehewitt15 — APPROVED** [review]
These changes look good

**Comment by mariacarmina** [comment]
Before merging I will make sure that the requests work. Thanks!

**Comment by mariacarmina** [comment]
I have tested right now with postman the routes and it seem to work from my side.
Screenshot from 2022-11-23 17-39-08
With which DID have you tested @jamiehewitt15?

**Comment by jamiehewitt15** [comment]
> DID have you tested @jamiehewitt15?

I just copy and pasted the examples that we have on the [aquarius API page](https://docs.oceanprotocol.com/api-references/aquarius-rest-api)

Screenshot from 2022-11-23 20-03-45

**Comment by mariacarmina** [comment]
> > DID have you tested @jamiehewitt15?
> 
> 
> 
> I just copy and pasted the examples that we have on the [aquarius API page](https://docs.oceanprotocol.com/api-references/aquarius-rest-api)
> 
> 
> 
> Screenshot from 2022-11-23 20-03-45
> 
> 
> 
> 

That's strange because from postman I didn't see any issues with it. I will investigate maybe we need to change the DID from the example.

**Comment by mariacarmina** [comment]
@jamiehewitt15 I have added a new DID in the examples for aquarius, these work now:

```console
curl --location --request POST 'https://v4.aquarius.oceanprotocol.com/api/aquarius/assets/names' \
>     --header 'Content-Type: application/json' \
>     --data-raw '{
>         "didList" : ["did:op:cd086344c275bc7c560e91d472be069a24921e73a2c3798fb2b8caadf8d245d6"]
>     }'
{"did:op:cd086344c275bc7c560e91d472be069a24921e73a2c3798fb2b8caadf8d245d6":"Ocean CEX Aggregator: OHLC history for OCEAN/USDT "}

```

File: api-references/aquarius-rest-api.md [hunk 1]
```diff
@@ -78,7 +78,7 @@
     curl --location --request POST 'https://v4.aquarius.oceanprotocol.com/api/aquarius/assets/names' \
```
Comment (jamiehewitt15) [inline api-references/aquarius-rest-api.md:78]: I got the response below, let me know if I'm doing anything wrong?

```
curl --location --request POST 'https://v4.aquarius.oceanprotocol.com/api/aquarius/assets/names' --header 'Content-Type: application/json'  --data-raw '{"didList" : ["did:op:CbD7aeecB5DFbABaB9126B5Cf1262dCFBA178479"] "didList" : ["did:op:cd086344c275bc7c560e91d472be069a24921e73a2c3798fb2b8caadf8d245d6"]}'

<!doctype html>
<html lang=en>
<title>400 Bad Request</title>
<h1>Bad Request</h1>
<p>The browser (or proxy) sent a request that this server could not understand.</p>

```

Comment (mariacarmina) [reply api-references/aquarius-rest-api.md:78]: Your DID has capital letters, I tested with `did:op:cd086344c275bc7c560e91d472be069a24921e73a2c3798fb2b8caadf8d245d6` and the console output response was 200 OK.

Comment (jamiehewitt15) [reply api-references/aquarius-rest-api.md:78]: Yeah works fine. Thanks a lot

**Review by jamiehewitt15 — APPROVED** [review]

**Pull request merged**